src/ui/components/Header/UserOptions.tsx

```diff
@@ -74,21 +74,21 @@ function UserOptions({ setModal, noBrowserItem }: UserOptionsProps) {
         orientation="bottom"
       >
         <button className="row" onClick={onDocsClick}>
-          <Icon iconName="docs" />
+          <Icon filename="docs" width="20px" height="20px" />
           <span>Docs</span>
         </button>
         <button className="row" onClick={onChatClick}>
-          <Icon iconName="help" />
+          <Icon filename="help" width="20px" height="20px" />
           <span>Chat with us</span>
         </button>
         <button className="row" onClick={onSettingsClick}>
-          <Icon iconName="settings" />
+          <Icon filename="settings" width="20px" height="20px" />
           <span>Settings</span>
         </button>
         {features.launchBrowser ? (
           window.__IS_RECORD_REPLAY_RUNTIME__ || noBrowserItem ? null : (
             <button className="row" onClick={onLaunchClick}>
-              <Icon iconName="replay" />
+              <Icon filename="replay-logo" width="20px" height="20px" />
```

Comment:
If all `<Icon />` are all supposed to be 20x20 for now, let's just hard code it into the component. 

Once we want to extend it to different sizes, it'd be nicer to just be able to pass some well-typed size (e.g. <Icon size="some-size" />, where it's `sm` | `md` | `lg` or whichever scale you prefer).